json/writer.go

```diff
@@ -23,3 +23,18 @@ func Write(doc common.AnyDocument, w io.Writer) error {
 
 	return nil
 }
+
+// WriteMultiline takes an SPDX Document and an io.Writer, and writes the document to the writer in JSON format with indents (multiline format).
+func WriteMultiline(doc common.AnyDocument, w io.Writer) error {
```

Comment:
i thought about this way, but i was not sure if you wanted to add arguments to `Write()` function, so that the other `Write()` functions ( i mean yaml, json, etc...) have same format.

> What would you think about this approach?

I think your solution is better. Tell me if this solution works for you - I'll update this PR.

Reply:
Since adding options as var args allows users to continue to use the same signature they already have been: `json.Write(doc, writer)`, I think this stays consistent with the other Write functions. If you wanted to add analogous options for `yaml.Write`, I wouldn't be opposed to that, but I don't think it's necessary.

Reply:
+1 on adding vargs which wouldn't change the interface, that would work nicely!

Reply:
Added changes in https://github.com/spdx/tools-golang/pull/213/commits/755da2841c60f5b6d3c5e4e9721997aece74d813